src/components/Patient/PatientRegistration.tsx

```
@@ -688,14 +689,34 @@ const PatientBasicsContent = ({
+                    facilityId={facilityId}
+                    selected={selectedTags}
+                    onChange={(tags) => {
+                      field.onChange(tags.map((tag) => tag.id));
+                    }}
+                    resource={TagResource.PATIENT}
+                  />
+                  {selectedTags.length > 0 && (
+                    <ul
+                      className="flex flex-wrap gap-2 mt-2 list-none"
+                      aria-label={t("selected_tags")}
+                      role="list"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<ol>
<ul>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)